src/modules/noit_lua_noit_binding.c

```diff
@@ -303,9 +225,16 @@ noit_message_index_func(lua_State *L) {
         break;
       }
       return 1;
+    case 'w':
+      if(!strcmp(k, "whence")) {
+        lua_pushinteger(L, msg->value.whence_ms/1000);
```

Comment:
I had the fear that working with large numbes (milliseconds) in lua could lead to rounding errors. But in the end 53 bit are more than enough for unix time in milliseconds. I'll change this to `whence_ms` again